### PR TITLE
cloud: add TiDB Cloud Dedicated Data Migration maintenance notification

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Add TiDB Cloud Dedicated Data Migration maintenance notification

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/notification-2024-04-18-dm-feature-maintenance.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+For clusters deployed on AWS:
+
+- During the upgrade, the DM tasks can keep running without disruption. The DM console can be used normally.
+
+For clusters deployed on Google Cloud:
+
+- The DM console will be unavailable for up to 30 minutes. During this period, you cannot create or manage DM tasks.
+- If a DM task is in the incremental migration stage, it will be interrupted for up to 30 minutes. During this period, do not purge the binary log of the MySQL database. The DM task will automatically resume after the upgrade is completed.
+- If a DM task is in the stage of exporting and importing full data, it will fail during the upgrade, and cannot be resumed after the upgrade. It is recommended not to create any DM task on the day when the upgrade is performed, to ensure that no DM tasks are in the stage of exporting and importing full data when the upgrade starts.
```
Comment (Frank945946) [inline tidb-cloud/notification-2024-04-18-dm-feature-maintenance.md:32]:
```suggestion

```

Comment (Frank945946) [reply tidb-cloud/notification-2024-04-18-dm-feature-maintenance.md:32]: 因为 4.18 日只会升级 AWS 日本 region ，GCP 的在 4.11 日都升级完成了，这里就不需要再提了。

**Review by Frank945946 — APPROVED** [review]

File: tidb-cloud/notification-2024-04-18-dm-feature-maintenance.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+During the maintenance window, the DM feature for TiDB Dedicated clusters in the following regions will be affected:
+
+- Cloud provider: AWS, region: Tokyo (ap-northeast-1)
+
+Apart from the DM feature in the TiDB cluster, all other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.
+
+For clusters deployed on AWS:
+
+- During the upgrade, the DM tasks can keep running without disruption. The DM console can be used normally.
+
```
Comment (Oreoxmt) [inline tidb-cloud/notification-2024-04-18-dm-feature-maintenance.md:27]:
```suggestion
```

File: TOC-tidb-cloud.md [hunk 3]
```diff
@@ -666,6 +666,8 @@
   - [2021](/tidb-cloud/release-notes-2021.md)
   - [2020](/tidb-cloud/release-notes-2020.md)
 - Maintenance Notification
+  - [[2024-04-18] TiDB Cloud Dedicated Data Migration (DM) Feature Maintenance Notification](/tidb-cloud/notification-2024-04-18-dm-feature-maintenance.md)
+  - [[2024-04-11] TiDB Cloud Dedicated Data Migration (DM) Feature Maintenance Notification](/tidb-cloud/notification-2024-04-11-dm-feature-maintenance.md)
```
Comment (hfxsd) [inline TOC-tidb-cloud.md:670]:
```suggestion
  - [[2024-04-18] TiDB Cloud Data Migration (DM) Feature Maintenance Notification](/tidb-cloud/notification-2024-04-18-dm-feature-maintenance.md)
  - [[2024-04-11] TiDB Cloud Data Migration (DM) Feature Maintenance Notification](/tidb-cloud/notification-2024-04-11-dm-feature-maintenance.md)
```

File: tidb-cloud/notification-2024-04-11-dm-feature-maintenance.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+---
+title: 2024-04-11 TiDB Cloud Dedicated Data Migration (DM) Feature Maintenance Notification
+summary: Learn about the details of TiDB Cloud Dedicated Data Migration (DM) feature maintenance on April 11, 2024, such as the maintenance window and impact.
+---
+
+# [2024-04-11] TiDB Cloud Dedicated Data Migration (DM) Feature Maintenance Notification
```
Comment (hfxsd) [inline tidb-cloud/notification-2024-04-11-dm-feature-maintenance.md:6]:
```suggestion
title: 2024-04-11 TiDB Cloud Data Migration (DM) Feature Maintenance Notification
summary: Learn about the details of TiDB Cloud Data Migration (DM) feature maintenance on April 11, 2024, such as the maintenance window and impact.
---

# [2024-04-11] TiDB Cloud Data Migration (DM) Feature Maintenance Notification
```

File: tidb-cloud/notification-2024-04-18-dm-feature-maintenance.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+---
+title: 2024-04-18 TiDB Cloud Dedicated Data Migration (DM) Feature Maintenance Notification
+summary: Learn about the details of TiDB Cloud Dedicated Data Migration (DM) feature maintenance on April 18, 2024, such as the maintenance window and impact.
+---
+
+# [2024-04-18] TiDB Cloud Dedicated Data Migration (DM) Feature Maintenance Notification
```
Comment (hfxsd) [inline tidb-cloud/notification-2024-04-18-dm-feature-maintenance.md:6]:
```suggestion
title: 2024-04-18 TiDB Cloud Data Migration (DM) Feature Maintenance Notification
summary: Learn about the details of TiDB Cloud Data Migration (DM) feature maintenance on April 18, 2024, such as the maintenance window and impact.
---

# [2024-04-18] TiDB Cloud Data Migration (DM) Feature Maintenance Notification
```

File: tidb-cloud/notification-2024-04-11-dm-feature-maintenance.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+- Cloud provider: AWS, region: Singapore (ap-southeast-1)
+- Cloud provider: AWS, region: Seoul (ap-northeast-2)
+- Cloud provider: AWS, region: Frankfurt (eu-central-1)
+- Cloud provider: AWS, region: São Paulo (sa-east-1)
+- Cloud provider: AWS, region: Oregon (us-west-2)
+- Cloud provider: Google Cloud, region: Oregon (us-west1)
+- Cloud provider: Google Cloud, region: Tokyo (asia-northeast1)
+- Cloud provider: Google Cloud, region: Singapore (asia-southeast1)
+
+Apart from the DM feature in the TiDB cluster, all other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.
```
Comment (hfxsd) [inline tidb-cloud/notification-2024-04-11-dm-feature-maintenance.md:31]:
```suggestion
The maintenance only affects the DM feature in the TiDB cluster. All the other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.
```

File: tidb-cloud/notification-2024-04-18-dm-feature-maintenance.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+- End time: 2024-04-18 09:00 (UTC+0)
+- Duration: 1 hour
+
+## Impact
+
+During the maintenance window, the DM feature for TiDB Dedicated clusters in the following regions will be affected:
+
+- Cloud provider: AWS, region: Tokyo (ap-northeast-1)
+
+Apart from the DM feature in the TiDB cluster, all other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.
```
Comment (hfxsd) [inline tidb-cloud/notification-2024-04-18-dm-feature-maintenance.md:22]:
```suggestion
The maintenance only affects the DM feature in the TiDB cluster. All the other functionalities remain unaffected. You can continue to manage the TiDB cluster and perform read/write operations or other operations as usual.
```

**Review by hfxsd — APPROVED** [review]

**Review by Frank945946 — APPROVED** [review]

**Pull request merged**